pyproject.toml

```diff
@@ -0,0 +1,126 @@
+[build-system]
+requires = ["setuptools>=64", "wheel", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "oculy"
+dynamic=["version"]
+description = "Visualisation app for Enaml based plugin applications"
+readme = "README.rst"
+requires-python = ">=3.7"
+license = {file = "LICENSE"}
+authors = [
+  {name = "Matthieu C. Dartiailh", email = "m.dartiailh@gmail.com"}
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+
+[options]
+packages = "find:"
+zip_safe = false
+long_description = "file: README.rst"
+long_description_content_type = "text/x-rst"
+platforms = "Linux; Windows; Mac"
+
+dependencies = [
+    "glaze",
+    "numpy",
+    "xarray",
+    "matplotlib",
+    "mplcairo",
+    "toml",
+    "typing_extensions; python_version<'3.10'"
+]
+use_2to3 = "False"
+
+
+[project.optional-dependencies]
+qt5-pyqt = ["enaml[qt5-pyqt]"]
+qt5-pyside = ["enaml[qt5-pyside]"]
+qt6-pyqt = ["enaml[qt6-pyqt]"]
+qt6-pyside = ["enaml[qt6-pyside]"]
```

Comment:
We can remove those per my above comment

Reply:
👍